bvm/ballerina-runtime/src/main/resources/MessagesBundle.properties

```diff
@@ -242,3 +242,5 @@ decimal.value.out.of.range = decimal range overflow
 config.size.mismatch = [{0}] the size for configurable variable ''{1}'' is expected to be ''{2}'', \
   but found ''{3}''
 invalid.utf8.byte.array.value = array contains invalid UTF-8 byte value
+negative.regexp.match.string.index = start index cannot be less than 0
+regexp.match.string.index.out.of.range = start index ''{0}'' cannot be greater than input length ''{1}''
```

Comment:
```suggestion
regexp.match.string.index.out.of.range = start index ''{0}'' cannot be greater than input string length ''{1}''
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/39238/commits/df4971a3ed0d730f8f55a69d712b7ecd2ff50979.